### DISCUSSION TITLE
Point Redemptions in Chat (Willing to donate money for it!)

### DISCUSSION THREAD

**Discussion by EmmaLlama** [baffler/Transparent-Twitch-Chat-Overlay#95] [Ideas]
Hihi, I love love love this software, it's so darn handy!

The only thing I wish is that there was a way to show point redemptions so I can react to those fast like I can with chat. I was wondering if there was any way to implement this; and if it will take a lot of work I'm definitely okay with paying for the devs' time to do so as it'd benefit my stream a huge amount.

Thank you again for creating this program for us all to use!

**Comment by baffler** [comment]
@EmmaLlama I think this should be possible using Twitch API. It's something I've already kinda worked with and may be able to get implemented into the app. If you're serious about the donation, you can DM me on twitter if you'd like, it's @baffler
